src/lib.rs

```diff
@@ -124,6 +125,44 @@ pub fn get_worker_id() -> usize {
 /// Socket used for coordinating a Gaggle distributed load test.
 pub(crate) struct Socket {}
 
+/// Wrapper over File object that contains context for errors
```

Comment:
Perhaps instead of moving this all into `lib.rs`, it would be more appropriate for it to live in `report.rs`?

Reply:
Good idea. Moved there